src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const movies = moviesFromServer;
```

Comment:
why not just use `moviesFromServer` in the component instead of assigning it to another const? 🤔